src/components/MultiselectProblem.tsx

```diff
@@ -1,7 +1,120 @@
+  correctResponse, encourageResponse, solution, retryLimit
+}: MultiselectProps): JSX.Element => {
+  const [feedback, setFeedback] = useState('')
+  const [formDisabled, setFormDisabled] = useState(false)
+  const [retriesAllowed, setRetriesAllowed] = useState(0)
+  const solutionArray: string[] = JSON.parse(solution)
+  const parsedOptionValues: string[] = JSON.parse(solutionOptions)
+
+  const schema = Yup.object({
+    response: Yup.array().min(1, 'Please select an answer')
+  })
+
+  const contentRefCallback = useCallback((node: HTMLDivElement | null): void => {
+    if (node != null) {
+      mathifyElement(node)
+    }
+  }, [])
+
+  const clearFeedback = (): void => {
+    setFeedback('')
+  }
+
+  const modifyModel = (values: MultiselectFormValues, e: React.ChangeEvent<HTMLInputElement>): string[] => {
+    const newSet = new Set(values.response)
+    if (e.target.checked) {
+      newSet.add(e.target.value)
+    } else {
+      newSet.delete(e.target.value)
+    }
+    return Array.from(newSet)
+  }
+
+  const generateOptions = (values: MultiselectFormValues, isSubmitting: boolean, setFieldValue: (field: string, value: any, shouldValidate?: boolean) => void): JSX.Element[] => {
+    const options: JSX.Element[] = []
+
+    parsedOptionValues.forEach(val => options.push(
+      <>
+    <label className="form-check-label" key={val}>
+      <Field
+      class="form-check-input"
+      type="checkbox"
+      name="response"
+      onChange={(e: React.ChangeEvent<HTMLInputElement>) => { clearFeedback(); setFieldValue('response', modifyModel(values, e)) }}
+      disabled={isSubmitting || formDisabled}
+      value={val}>
+      </Field>
+      {' ' + val}
+    </label>
+    <br></br>
+    </>
+    ))
+
+    return options
+  }
+
+  const compareForm = (form: string[], solution: string[]): boolean => {
+    if (form.length !== solution.length) {
+      return false
+    }
+    for (let i = 0; i < solution.length; i++) {
+      if (!solution.includes(form[i])) {
+        return false
+      }
+    }
+    return true
+  }
+
+  const handleSubmit = async (values: MultiselectFormValues): Promise<void> => {
+    if (compareForm(values.response, solutionArray)) {
+      setFeedback(correctResponse)
+      solvedCallback()
+      setFormDisabled(true)
+    } else if (retryLimit === 0 || retriesAllowed !== retryLimit) {
+      setRetriesAllowed(currRetries => currRetries + 1)
+      setFeedback(encourageResponse)
+      allowedRetryCallback()
+    } else {
+      exhaustedCallback()
+      setFeedback(NO_MORE_ATTEMPTS_MESSAGE)
+      setFormDisabled(true)
+    }
+  }
 
-export const MultiselectProblem = ({ solvedCallback, exhaustedCallback, allowedRetryCallback }: MultiSelectProps): JSX.Element => {
-  return <></>
+  return (
+    <div className="os-raise-bootstrap" ref={contentRefCallback}>
+      <div className="form-check my-3" dangerouslySetInnerHTML={{ __html: content }} />
```

Comment:
You don't want to put `form-check` here. It causes the spacing of the problem to be off. See corresponding fix to styling above.